docs/mkdocs/secureboot-signing.md

```diff
@@ -0,0 +1,75 @@
+# Signing kernel modules using KMM
+
+### Secure-boot with KMM
+
+For more details on using Secure-boot see [here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_monitoring_and_updating_the_kernel/signing-kernel-modules-for-secure-boot_managing-monitoring-and-updating-the-kernel) or [here](https://wiki.debian.org/SecureBoot)
+
+### Using Signing with KMM
+
+On a secure-boot enabled system all kernel modules (kmods) must be signed with a public/private key-pair enrolled into the Machine Owner's Key or MOK database. Drivers distributed as part of a distribution should already be signed by the distributions private key, but for kernel modules build out-of-tree KMM supports signing kernel modules using the "sign" section of the kernel mapping.
+
+To use this functionality you need:
+
+ * A public private key pair in the correct (der) format
+ * At least one secure-boot enabled nodes with the public key enrolled in its MOK database
+ * A driver container image 
+
+The yaml below will add the public/private key-pair as secrets with the required key names ("key" for the private key, "cert" for the public key). It will then pull down the `unsignedImage` image, open it up, sign the kernel modules listed in `filesToSign`, add them back and push the resulting image as `containerImage`. 
+
+KMM should then deploy the daemonset that loads the signed kmods onto all the nodes with that match the selector.
+The driver containers should run successfully on any nodes that have the public key in their MOK database, and any nodes that are not secure-boot enabled (which will just ignore the signature). They should fail to load on any that have secure-boot enabled but do not have that key in their MOK database.
+
+If your driver containers end up in `PostStartHookError` or `CrashLoopBackOff` status, and `kubectl describe` shows an event: `modprobe: ERROR: could not insert '<your kmod name>': Required key not available` then the kmods are either not signed, or signed with the wrong key.
```

Comment:
How about having a shared `Debugging` headline for the 2 examples instead of duplicating it?